tokens/functional/shadow/light.json5

```diff
@@ -0,0 +1,100 @@
+{
+  shadow: {
+    color: {
+      small: {
+        $value: '{base.color.scale.black}',
+        alpha: 0.96,
+        $type: 'color'
+      },
+      medium: {
+        $value: '{base.color.scale.gray.4}',
+        alpha: 0.85,
+        $type: 'color'
+      },
+      large: {
+        $value: '{base.color.scale.gray.4}',
+        alpha: 0.8,
+        $type: 'color'
+      },
+      extraLarge: {
+        $value: '{base.color.scale.gray.4}',
+        alpha: 0.7,
+        $type: 'color'
+      },
+      highlight: {
+        $value: '{base.color.scale.white}',
+        alpha: 0.25,
+        $type: 'color'
+      },
+      inset: {
+        $value: '{base.color.scale.gray.2}',
+        alpha: 0,
+        $type: 'color'
+      },
+      semiBlack: {
+        "12": {
```

Comment:
Should this be nested here? and what does "12" mean here?

Reply:
Yes, this is a `semiBlack` with 12% opacity. But this is all WIP as the shadows need rework: https://github.com/primer/primitives/issues/359